Synchronizer.js

```diff
@@ -44,7 +44,7 @@ rOneTimeCode - One time pass code from Remarkable that can typically
 gdFolderSearchParams - Google Drive search SDK string or folder id.
 rRootFolderName - The root folder in Remarkable device. Currently this
                   must already exist on your device. This can be a remarkable
-                  folder GUID if you know it.
+                  folder GUID if you know it. Use False for Remarkable root.
```

Comment:
Setting root to false likely introduces a bug in `rAllDescendantIds` (when walking the document list, no folder has a parent).